### ISSUE TITLE
AngularJS 2, TypeScript and Angular-Chart

### ISSUE THREAD

**Issue by anderflash** [jtblin/angular-chart.js#349]
Hi, 
Is there any plans to use AngularJS 2? For example, with Typescript.
Thanks

**Comment by jtblin** [comment]
Yes. I was planning to have a look after support for chart.js 2.0 is merged which should happen pretty soon, see https://github.com/jtblin/angular-chart.js/tree/chartjs-2.0.

**Comment by johnmanko** [comment]
I'm keeping an eye on this. :) Do we have an estimated ETA for the Chart.js 2.0 merger, and then support for AngularJS 2?

**Comment by jtblin** [comment]
Chart.js 2.x merge should be coming soon. I've added support for all new types of charts and got pretty much all the breaking changes in that I wanted so far. I'm thinking of publishing as beta (i.e. no more changes to the API) in the next few days, bake for 1 or 2 weeks then merge to master. I'll probably start converting to typescript soon after that, then support for Angular 2.x.

**Comment by atorefrank** [comment]
Will you be posting here as soon as typescript is enabled?

**Comment by jtblin** [comment]
Yes, I will be posting the progress here. Not sure when I'm going to start on that yet.

**Comment by jtblin** [comment]
I've started to look at typescript conversion over the weekend however it's going to take some time...

**Comment by elekzalan** [comment]
Hello @jtblin ! Any news on this topic?

**Comment by jtblin** [comment]
This project is specifically for AngularJS (1.x). For modern Angular (2+), please look at other libraries like `ng2-charts`. We are, however, currently modernizing the internal codebase of this library to use TypeScript and upgrading to Chart.js v4.
